exp-player/addon/components/exit-warning.js

```diff
@@ -0,0 +1,20 @@
+import Ember from 'ember';
+import layout from '../templates/components/exit-warning';
+
+export default Ember.Component.extend({
+    layout,
+    warn: false,
+    exitReason: null,
+    exitPrivacy: null,
+    actions: {
+        onConfirm() {
+            this.get('onConfirm')({
```

Comment:
Any reason to not just let the actions bubble up here? Or just call `this.send` or  `this.sendAction`